File: ILT.IHR.DTO/WFH.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ILT.IHR.DTO
{
    public partial class WFH : AbstractDataObject
    {
        public int WFHID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        [Required]
        public int? EmployeeID { get; set; }
        public string EmployeeName { get; set; }
        [Required]
        public string Title { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int RequesterID { get; set; }
        public string Requester { get; set; }
        public int? ApproverID { get; set; }
        public string Approver { get; set; }
        public int StatusID { get; set; }
        public string StatusValue { get; set; }
        public string Status { get; set; }
        public string Comment { get; set; }
        public Guid LinkID { get; set; }
    }
}

File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/WFH.ts
export class IWFH {
    WFHID: number;
    EmployeeID: number;
    Title: string;
    StartDate: Date;
    EndDate: Date;
    RequesterID: number;
    ApproverID?: number;
    StatusID: number;
    EmployeeName: string;
    LinkID: string;
    Comment: string;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy: string;
    ModifiedDate: Date;
    TimeStamp: ['']
}
export class IWFHDisplay {
    wfhid: number;
    employeeID: number;
    employeeName: string;
    requesterID: number;
    requester: string;
    approverID: number;
    approver: string;
    statusID: number;
    statusValue: string;
    status: string;
    title: string;
    comment: string;
    startDate: Date;
    endDate: Date;
    linkID: string;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}
